monitorables/gitlab/api/models/issuesparams.go

```diff
@@ -0,0 +1,21 @@
+//+build !faker
+
+package models
+
+import "github.com/monitoror/monitoror/internal/pkg/monitorable/params"
+
+type (
+	IssuesParams struct {
+		params.Default
+
+		ProjectID *int `json:"projectId" query:"projectId"`
```

Comment:
Not required?

Reply:
According gitlab api documentation you can query issues with and without project ID:
- https://docs.gitlab.com/ee/api/issues.html#list-issues
- https://docs.gitlab.com/ee/api/issues.html#list-project-issues

Reply:
OK :)